src/vim9expr.c

```diff
@@ -639,6 +639,68 @@ generate_funcref(cctx_T *cctx, char_u *name, int has_g_prefix)
     return generate_PUSHFUNC(cctx, ufunc->uf_name, ufunc->uf_func_type, TRUE);
 }
 
+/*
+ * Look for ".member", check if a member of the specified class.
+ *
+ * If found, fill in the information about the member and the parsing;
+ * and return TRUE. Some information may be saved even when return FALSE.
+ *
```

Comment:
@yegappan This commit is to optimize something that probably doesn't belong in the spec, and will probably be closed without merge.

But the `find_class_object_member()` might be useful. It's like a "peek at next token, return true if a member of the given class". It provides enough input stream info so the caller can consume the token if wanted.

Could get rid of everything in this PR, except for that function; change the title. Then it's a pending PR with something for the toolkit when/if needed. Probably cleaner to just open another PR.